### ISSUE TITLE
New `schema` initOption is not included in the type definitions

### ISSUE THREAD

**Issue by Vinnl** [vitaly-t/pg-promise#507]
### Expected behavior

The TypeScript compiler not to complain when I configure the `schema` option when initialising pg-promise.

### Actual behavior

The TypeScript compiler complains:

```
error TS2322: Type '{ schema: string | undefined; }' is not assignable to type 'IOptions<{}>'.
Object literal may only specify known properties, and 'schema' does not exist in type 'IOptions<{}>'.
```

### Steps to reproduce

    import * as pgPromise from 'pg-promise';
    const initOptions: pgPromise.IOptions<{}> = { schema: 'some-schema' };

### Environment

* Version of pg-promise: 8.3.0
* OS type Linux/Windows/Mac: Linux
* Version of Node.js: 8.11.1

**Comment by vitaly-t** [comment]
Ouch, how did I forget.... :smile:

**Comment by vitaly-t** [comment]
Re-opening after ayto-close. I usually close issues after a new release that contains the fix.

**Comment by vitaly-t** [comment]
Fixed in [v8.3.1](https://github.com/vitaly-t/pg-promise/releases/tag/v.8.3.1).

**Comment by vitaly-t** [comment]
[Version 8.3.2](https://github.com/vitaly-t/pg-promise/releases/tag/v.8.3.2) further improves on this feature, with support of callbacks.
